ast/builtins.go

```diff
@@ -1254,7 +1254,8 @@ var RandIntn = &Builtin{
 		),
 		types.Named("y", types.N).Description("random integer in the range `[0, abs(n))`"),
 	),
-	Categories: number,
+	Categories:       number,
+	Nondeterministic: true,
```

Comment:
🤔 I wonder if this is the best word...  Or rather, I think it's a good choice, but _not_ annotating the JWT-related builtins doesn't feel right: They are not deterministic, but we're saying they are _not_ for other reasons. Perhaps we should add another field, or some other array of "sensitive" exceptions...?

Reply:
I discussed this with @tsandall the other day, and IIRC the conclusion we came to was that we should record everything by default, but we'll need to provide an option in the log-scrubbing/censoring tooling to clean up the ND builtins cache records for the JWT-related builtins (to prevent keys from being recorded, etc.).

Reply:
I have added the JWT builtins to the ND list, and added some security-related (masking policy) tasks for myself.